scripts/checkForOrphanImports.js

```diff
@@ -31,7 +31,29 @@ const getAllImageImports = (path) => {
         return null;
       }
       const textfile = fs.readFileSync(file, 'utf-8');
-      const imports = textfile.match(importRegex)?.map((importStatement) => {
+      const fullImportStatements = textfile.match(importRegex);
+      if (fullImportStatements?.length > 1) {
+        const lines = textfile.split('\n');
+        for (let i = 0; i < fullImportStatements.length - 1; i++) {
+          const importLine1 = lines.indexOf(fullImportStatements[i]);
+          const importLine2 = lines.indexOf(fullImportStatements[i + 1]);
+          if (importLine2 - importLine1 <= 1) {
+            console.log(
+              `\n The imports on lines \x1b[31m${file}:${
+                importLine1 + 1
+              }\x1b[0m and \x1b[31m${
+                importLine2 + 1
+              }\x1b[0m do not have adequate spacing`
```

Comment:
could we update this to explicitly mention a new line is needed?
```suggestion
              }\x1b[0m need a new line between them`
```